### ISSUE TITLE
loss and acc don't change during each epoch

### ISSUE THREAD

**Issue by michealshi23** [fchollet/keras#2392]
## **my code:**

```
model = Sequential()
model.add(Convolution1D(nb_filter=16,filter_length=10, init='uniform',border_mode='valid',activation='relu',subsample_length=1,input_dim=input_dim,input_length=input_length))
model.add(MaxPooling1D(pool_length=2))
model.add(Convolution1D(nb_filter=32,filter_length=8, init='uniform',border_mode='valid',activation='relu',subsample_length=1))
model.add(MaxPooling1D(pool_length=2))
model.add(Flatten())
model.add(Dense(hidden_dims))
model.add(Dropout(0.25))
model.add(Activation('relu'))
model.add(Dense(1))
model.add(Activation('sigmoid'))
model.compile(loss='binary_crossentropy',optimizer=Adagrad(lr=1.0, rho=0.95, epsilon=1e-06),class_mode='binary')
model.fit(X_train, Y_train, batch_size=128, nb_epoch=100, verbose=1, callbacks=[],validation_split=0.0, validation_data=None, shuffle=True, show_accuracy=True, class_weight=None, sample_weight=None)
```
## **this is my training log:**

Epoch 1/100
89595/89595 [==============================] - 9s - loss: 5.7178 - acc: 0.6446  
Epoch 2/100
89595/89595 [==============================] - 8s - loss: 5.7257 - acc: 0.6448  
Epoch 3/100
89595/89595 [==============================] - 8s - loss: 5.7257 - acc: 0.6448  
Epoch 4/100
89595/89595 [==============================] - 8s - loss: 5.7257 - acc: 0.6448  
Epoch 5/100
89595/89595 [==============================] - 8s - loss: 5.7257 - acc: 0.6448  
Epoch 6/100
89595/89595 [==============================] - 8s - loss: 5.7257 - acc: 0.6448 

it's on GPU: Using gpu device 0: GeForce GTX 970

**Comment by EnriqueSMarquez** [comment]
Check you are not shuffling the training targets or inputs. Normally that happens when the labels do not correspond with the inputs.

**Comment by michealshi23** [comment]
I  thought each epoch is trained based on the previous one, isn't that so ?
